src/nft/components/sell/select/SelectPage.tsx

```diff
@@ -1,3 +1,380 @@
+  }, [ownerAssets, setWalletAssets])
+
+  useEffect(() => {
+    ownerCollections && setWalletCollections(ownerCollections)
+  }, [ownerCollections, setWalletCollections])
+
+  const listFilter = useWalletCollections((state) => state.listFilter)
+
+  useEffect(() => {
+    if (searchText) {
+      const filtered = walletAssets.filter((asset) => asset.name?.toLowerCase().includes(searchText.toLowerCase()))
+      setDisplayAssets(filtered, listFilter)
+    } else {
+      setDisplayAssets(walletAssets, listFilter)
+    }
+  }, [searchText, walletAssets, listFilter, setDisplayAssets])
+
+  useEffect(() => {
+    setDisplayAssets(walletAssets, listFilter)
+  }, [walletAssets, listFilter, setDisplayAssets])
+
+  useEffect(() => {
+    if (ownerCollections?.length && collectionStats?.length) {
+      const ownerCollectionsCopy = [...ownerCollections]
+      for (const collection of ownerCollectionsCopy) {
+        const floorPrice = collectionStats.find((stat) => stat.address === collection.address)?.floorPrice
+        collection.floorPrice = floorPrice ? Math.round(floorPrice * 1000 + Number.EPSILON) / 1000 : 0 //round to at most 3 digits
+      }
+      setWalletCollections(ownerCollectionsCopy)
+    }
+  }, [collectionStats, ownerCollections, setWalletCollections])
+
+  return (
+    // Column style is temporary while we move over the filters bar that adjust width
+    <Column style={{ width: 'calc(100vw - 32px)' }}>
+      <Row
+        alignItems="flex-start"
+        position="relative"
+        paddingLeft={{ sm: '0', md: '52' }}
+        paddingRight={{ sm: '0', md: '72' }}
+        paddingTop={{ sm: '16', md: '40' }}
+      >
+        <AnimatedBox paddingX="16" flexShrink="0" width="full">
+          <Row gap="8" flexWrap="nowrap">
+            <CollectionSearch searchText={searchText} setSearchText={setSearchText} />
+            <SelectAllButton />
+          </Row>
+          <InfiniteScroll
+            next={fetchNextPage}
+            hasMore={hasNextPage ?? false}
+            loader={
+              hasNextPage ? (
+                <Center>
+                  <LoadingSparkle />
+                </Center>
+              ) : null
+            }
+            dataLength={displayAssets.length}
+            style={{ overflow: 'unset' }}
+          >
+            <div className={assetList}>
+              {displayAssets && displayAssets.length
+                ? displayAssets.map((asset, index) => <WalletAssetDisplay asset={asset} key={index} />)
+                : null}
+            </div>
+          </InfiniteScroll>
+        </AnimatedBox>
+      </Row>
+      {sellAssets.length > 0 && (
+        <Row
+          display={{ sm: 'flex', md: 'none' }}
+          position="fixed"
+          bottom="60"
+          left="16"
+          height="56"
+          borderRadius="12"
+          paddingX="16"
+          paddingY="12"
+          style={{ background: '#0d0e0ef2', width: 'calc(100% - 32px)', lineHeight: '24px' }}
```

Comment:
should color be added to the styles file? also notice the same `-32px` logic here so wanna know if that's related to above

Reply:
This color is a one-off only used in this instance

Reply:
Yes the width logic is also related to a temporary fix to make the content more viewable while waiting for the filters menu to be added to the page